runtime/context.go

```diff
@@ -54,3 +54,7 @@ func (c *Context) InitializeCodesAndPrograms() {
 		c.programs = map[common.LocationID]*ast.Program{}
 	}
 }
+
+func (c Context) UseMemory(usage common.MemoryUsage) {
```

Comment:
Whats the usage of this function?

Reply:
It's a leftover from before; I can remove it now I think.